debian/rules

```diff
@@ -8,6 +8,7 @@ export DEB_LDFLAGS_MAINT_APPEND = -Wl,-z,defs -Wl,--as-needed
 
 override_dh_auto_configure:
 	dh_auto_configure -- \
+		--libexecdir=/usr/lib/gnome-initial-setup \
```

Comment:
I could take this or leave it… I'm reasonably sure nothing outside gnome-initial-setup relies on this path and it's one more bit of divergence to carry forever, but OTOH it's not a big patch to carry.

Reply:
OK, the only thing I can find inside gnome-initial-setup is [this](https://github.com/endlessm/gnome-initial-setup/blob/master/data/20-gnome-initial-setup.rules#L27), which should be deleted anyway as that script doesn't seem to exist. I'm a bit concerned that changing this might invalidate all the testing I've done so far, since it seems like the likely failure points would be in the "unusual" modes such as demo mode...

Reply:
That script was a precursor to the `eos-transient-setup.service` systemd unit. Indeed it should go away.

OTOH, https://github.com/endlessm/eos-event-recorder-daemon/pull/234 would be such a place outside gnome-initial-setup :-)